broker-plugins/logging-logback/pom.xml

```diff
@@ -47,6 +47,11 @@
       <artifactId>logback-classic</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>de.siegmar</groupId>
+      <artifactId>logback-gelf</artifactId>
+    </dependency>
```

Comment:
I'll leave detailed review/discussion for those more familiar with the broker and its logging bits, but one key thing sticks out here that affects things to some extent:

The logback-gelf dependency is LGPL 2.1 licensed which means it is considered Category X (https://www.apache.org/legal/resolved.html#category-x) and cant be distributed by the Qpid project, for example within the broker release convenience binary, or be required for the broker to operate in typical usage, but could potentially be used in optional functionality.